docs/overview/what_is__-__main_areas/con-tracing.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I think we could merge this file with "Tracing with Jaeger" - both are introductions about how tracing works in Kyma, and both are short enough so they don't really need to be split up